app/workers/audio_callback_worker.rb

```diff
@@ -63,15 +74,84 @@ def update_audio(audio, job)
       audio.status_message = nil
     end
 
-    # save and announce the audio changes on its story
-    end_state = audio.status_message ? "#{audio.status} => #{audio}" : audio.status
-    Shoryuken.logger.info("Updating #{job['type']}[#{audio.id}]: status => #{end_state}")
-    audio.save!
+    finish(audio)
   end
+  # rubocop:enable Metrics/AbcSize, Metrics/PerceivedComplexity
 
   private
 
+  def finish(audio)
+    # save and announce the audio changes on its story
```

Comment:
Wait, where's the "announce"?  It seems to be up in `perform()` ... not down here.  (The old comment also looks inaccurate).

Reply:
ah yes, this is from before when we were not announcing after every callback, will fix.